pkg/config/configobserver/secretconfigobserver_test.go

```diff
@@ -109,14 +97,14 @@ func Test_ConfigObserver_ChangeSupportConfig(t *testing.T) {
 			},
 			expErr: nil,
 		},
-		{name: "set-all-config",
+		{
+			name: "set-all-config",
 			config: map[string]*corev1.Secret{
 				pullSecretKey: {Data: map[string][]byte{
-					".dockerconfigjson": nil,
+					".dockerconfigjson": []byte("{\"auths\":{\"cloud.openshift.com\":{\"auth\":\"testtoken\",\"email\":\"test\"}}}"),
```

Comment:
nit: you could use here -> ` quotes to not having to escape every double quote
I didn't test it with []byte casting though, but it works in regexp for example